packages/wallets/src/metamask/pages/settings.page.ts

```diff
@@ -0,0 +1,77 @@
+import { Locator, Page, test } from '@playwright/test';
+import { WalletConfig } from '../../wallets.constants';
+
+export class SettingsPage {
+  page: Page;
+  tabBarMenu: Locator;
+  networksTabButton: Locator;
+  experimentalTabButton: Locator;
+  addNetworkButton: Locator;
+  addNetworkManuallyButton: Locator;
+  networkNameInput: Locator;
+  networkRpcUrlInput: Locator;
+  networkChainIdInput: Locator;
+  networkTickerInput: Locator;
+  networkExplorerUrlInput: Locator;
+  saveNewTokenButton: Locator;
+  selectNetworksForEachSiteToggleState: Locator;
+  selectNetworksForEachSiteToggle: Locator;
+
+  constructor(
+    page: Page,
+    private extensionUrl: string,
+    public config: WalletConfig,
```

Comment:
private?

Reply:
it was `private` before refactoring in the `MetamaskPage class`